app/models/student.py

```diff
@@ -12,10 +12,16 @@
 from sqlalchemy import desc
 from sqlalchemy.ext import hybrid
 
-
+studentFields={
+    'sid':fields.Integer,
+    'name':fields.String,
+    'clas':fields.Integer,
+    'division':fields.String
+}    
 
 class SMS(db.Model):
```

Comment:
Class name should be descriptive. @siraj-shaikh-BS

Reply:
I tried it with Student, but it's giving me the error.